### ISSUE TITLE
Export E57 with metadata fails

### ISSUE THREAD

**Issue by cnovel** [cloudcompare/trunk#359]
Hello, I'm loading an e57 file in CloudCompare in order to segment it. Once it's done, I save the segmented result as E57. When reading it again, some metadata (pose + rotation) are missing. Do you have any idea why? Dataset can be found [here](http://sourceforge.net/projects/e57-3d-imgfmt/files/E57Example-data/pumpNoInvalidPoints.e57/download). Thanks.

**Comment by cloudcompare** [comment]
Indeed, at loading time parts of E57 meta data are dropped because they have no equivalent in CloudCompare. And same thing, when a cloud it saved as E57 (it could be any cloud by the way), we don't create fields for which we have no info.

For the particular case of pose and rotation, it is taken into account when loading the scans but it's integrated in the resulting points coordinates. It is planned to save this information in a 'sensor' structure (it's in the pipe, but it hasn't been done yet afaik).

**Comment by cnovel** [comment]
Thanks for the quick response. So almost every information in the "File Structure" folder is dropped ? For what I can see only Intensity limits are saved.

**Comment by cloudcompare** [comment]
Maybe a few others if you are lucky ;)
